### PR TITLE
Make global capacity and global rewards ratios constant

### PR DESCRIPTION
**Pull request by shark0der**
## Context
Reduce Cover contract size further

## Changes proposed in this pull request
Make GlobalCapacity and GlobalRewards ratios constants

## Test plan
Existing tests pass

## Checklist

- [x] Rebased the base branch
- [x] Attached corresponding Github issue
- [x] Prefixed the name with the type of change (i.e. feat, chore, test)
- [x] Performed a self-review of my own code
- [x] Followed the style guidelines of this project
- [ ] Made corresponding changes to the documentation
- [x] Didn't generate new warnings
- [x] Didn't generate failures on existing tests
- [ ] Added tests that prove my fix is effective or that my feature works

## Review

When reviewing a PR, please indicate intention in comments using the following emojis:
* :cake: = Nice to have but not essential.
* :bulb: = Suggestion or a comment based on personal opinion
* :hammer: = I believe this should be changed.
* :thinking: = I don’t understand something, do you mind giving me more context?
* :rocket: = Feedback

### CODE REVIEW

File: contracts/modules/cover/Cover.sol [hunk 1]
```diff
@@ -56,6 +53,9 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
 
   /* ========== CONSTANTS ========== */
 
+  uint private constant GLOBAL_CAPACITY_RATIO = 20000;
```
Comment (danoctavian) [inline contracts/modules/cover/Cover.sol:56]: add the clarfying comment here // x2

File: contracts/modules/cover/Cover.sol [hunk 2]
```diff
@@ -56,6 +53,9 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
...
+  uint private constant GLOBAL_REWARDS_RATIO = 5000;
```
Comment (danoctavian) [inline contracts/modules/cover/Cover.sol:57]: add the clarifying comment here // 50%

**Review by danoctavian — APPROVED** [review]
LGTM, those comments would be nice

File: contracts/modules/cover/Cover.sol (hunk 2 above)
Comment (roxdanila) [reply contracts/modules/cover/Cover.sol]: done

**Pull request merged**